### DISCUSSION TITLE
Leaderboard Tickets (Bug Reporting)

### DISCUSSION THREAD

**Discussion by Keltron3030** [RetroAchievements/RAWeb#763] [Ideas]
As a developer with an interest in leaderboards I would like to see a bug reporting subset for leaderboards.  This way interested developers can easily seek out and fix broken boards.

 As a starting point a query of leaderboard comments could be created looking for keywords like "broken", "fix", "bug", "glitch" to auto create a working list of potentially bugged leaderboards.

To widen the net we could increase the query to also look for low score leaderboards where the top 3 entries are zero or high score leaderboards where the top 3 entries are identical as a way to seek out unreported bugs.

**Comment by kdecks** [comment]
Related: #51
